### PR TITLE
Show data even when we are not cheaper

### PR DESCRIPTION
**Pull request by lonamiaec**
Avoid showing an empty table when we are not the cheapest provider

### CODE REVIEW

File: src/comparison-table/ComparisonTable.js [hunk 1]
```diff
@@ -90,6 +90,11 @@ export default class ComparisonTable extends Component {
     );
   }
 
+  transferwiseIsTheCheapestProvider() {
+    debugger;
```
Comment (daves125125) [inline src/comparison-table/ComparisonTable.js]: remove?

File: src/comparison-table/ComparisonTable.js [hunk 2]
```diff
@@ -128,6 +133,13 @@ export default class ComparisonTable extends Component {
             return <CardProvider key={p.id} name={p.name} logo={p.logo} fees={p.fees} rate={p.rate} hiddenFees={p.hiddenFees} maxFee={p.maxFee} source={this.props.source} target={this.props.target} amount={p.amount} highestAmount={providers[0].amount} collectedAt={p.collectedAt}/>
           })}
         </div>
+        {this.transferwiseIsTheCheapestProvider() &&
+        <div className="m-b-2">
+          <div className="alert alert-info" role="alert">
+            We're always honest with our customers. And honestly, you'd be better off using another provider this time. But bear in mind that might be because they're running a promotion or special offer. <a href="">Find out more here.</a>
```
Comment (daves125125) [inline src/comparison-table/ComparisonTable.js]: The message / template in this file ComparisonTable.js and in ComparisonBarTable.js look duplicated? 

Opportunity for reuse by pulling into a common template?

File: src/ComparisonApi.js [hunk 3]
```diff
@@ -13,9 +13,9 @@ function request(path) {
 }
 
 export function comparisonsFor(source, target, amount) {
-  return request('/comparisons?source=' + source + '&target=' + target + '&amount=' + amount + '&includeGoogle=false');
+  return request('/comparisons?source=' + source + '&target=' + target + '&amount=' + amount + '&includeGoogle=false&excludeCheaper=false');
 }
 
 export function providersFor(source, target) {
-  return request('/comparisons/providers?source=' + source + '&target=' + target);
+  return request('/comparisons/providers?source=' + source + '&target=' + target +"&excludeCheaper=false");
```
Comment (daves125125) [inline src/ComparisonApi.js]: For my understanding, who still expects the excludeCheaper to be set to true? And is the plan to revert to the cheaper option by default in the future (assuming all widgets have the copy included)?

File: src/comparison-table/ComparisonTable.js (hunk 1 above)
Comment (lonamiaec) [inline src/comparison-table/ComparisonTable.js]: OMG I forgot this breakpoint.. 🤦‍♂️  The good thing is `npm run build` remove this things from production code 
Good catch!

File: src/comparison-table/ComparisonTable.js (hunk 2 above)
Comment (lonamiaec) [inline src/comparison-table/ComparisonTable.js]: Not at all... the two components use differents html elements to show the copy. The only thing we could extract here is the copy itself, and it'll be done for translations

File: src/ComparisonApi.js (hunk 3 above)
Comment (lonamiaec) [inline src/ComparisonApi.js]: The `excludeCheaper` param  to `False` is the only way to change the `comparison-service` behaviour to show data even when we are not the cheapest
It's a temporary fix, as we want to change the default behaviour, but it's not so easy, as we need to chat with some other teams to see if it's ok from their side or we need to change something else

**Pull request merged**